.github/workflows/terraform-create-repository.yml

```
@@ -251,6 +297,50 @@ jobs:
 
+      - name: Validate bootstrap inputs
+        id: validate-inputs
+        shell: bash
+        env:
+          LANGUAGES: ${{ inputs.languages }}
+          ENV_MANAGER: ${{ inputs.env_manager }}
+          PYTHON_VERSION: ${{ inputs.python_version }}
+          NODE_VERSION: ${{ inputs.node_version }}
+          GO_VERSION: ${{ inputs.go_version }}
+          JAVA_VERSION: ${{ inputs.java_version }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:27:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)